docs/scripting/what-is-script.md

```diff
@@ -4,10 +4,23 @@ category: General
 tags:
     - experimental
 mentions:
-	- cda94581
+	- ConsoleTerm
 ---
 
 ::: warning
-The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft beta 1.19.60.26
+The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft beta 1.19.70
 :::
 
+## API
+API is stands for (Application Programming Interface), it is a kind of establishing contacts between two applications, the term application contains a lot more than you might imagine under the term application, it can be a simple script or a game or an entire operating system. The API that you encounter the most is right in front of you, and that API is between the application and your operating system, for example with the cursor on the screen, why is it related to the API? Because your application can find out through the API where the cursor is on the screen without having to implement its own cursor, this means that Minecraft also uses a lot of Wrappers and they then use the API system directly so that we can see the application. Summary:
+An API is a kind of mutual or one-sided interaction between two or more Applications!
```

Comment:
Move the `Summary:` part into line 16.
`Summary: An API is a kind of mutual or one-sided interaction between two or more Applications!`